docs/extend/plugin_api.md

```diff
@@ -43,26 +33,26 @@ user or container tries to use one by name.
 
 There are three types of files which can be put in the plugin directory.
 
-* `.sock` files are UNIX domain sockets.
+* `.sock` files are Unix domain sockets.
```

Comment:
Ah! TIL `UNIX` is the trademarked name, and `Unix` is the "common" name ("Unix-like"); https://unix.stackexchange.com/a/532341/78656